np.sh

```diff
@@ -15,7 +15,7 @@ if test "0" != "$(git rev-list --count --left-only @'{u}'...HEAD)"; then
 	exit 128;
 fi
 
-trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
+trashCli=$(node -p "require.resolve('.bin/trash')");
```

Comment:
Actually, this won't work, as `__dirname` is `process.cwd()`/PWD, not the directory of `np.sh`.